src/DataGrid.tsx

```diff
@@ -127,12 +127,9 @@ export interface DataGridProps<R, SR = unknown, K extends Key = Key> extends Sha
   selectedRows?: ReadonlySet<K> | null;
   /** Function called whenever row selection is changed */
   onSelectedRowsChange?: ((selectedRows: Set<K>) => void) | null;
-  /** The key of the column which is currently being sorted */
-  sortColumn?: string | null;
-  /** The direction to sort the sortColumn*/
-  sortDirection?: SortDirection | null;
-  /** Function called whenever grid is sorted*/
-  onSort?: ((columnKey: string, direction: SortDirection) => void) | null;
+  /**Used for multi column sorting */
+  sortColumns?: readonly Readonly<SortColumn>[];
+  onSortColumnsChange?: (sortColumns: SortColumn[]) => void;
```

Comment:
Let's allow `null` as well (more context: https://github.com/adazzle/react-data-grid/pull/2406)
```suggestion
  sortColumns?: readonly Readonly<SortColumn>[] | null;
  onSortColumnsChange?: ((sortColumns: SortColumn[]) => void) | null;
```